packages/fxa-payments-server/server/lib/amplitude.js

```diff
@@ -130,6 +84,16 @@ module.exports = (event, request, data) => {
       }
     }
 
+    const dnt = request.headers?.['dnt'] === '1';
+    if (dnt) {
+      amplitudeEvent.event_properties = filterDntValues(
```

Comment:
Similar to another comment, are we sure these values are defined. If not, check here or check in filterDntValues function.